client/src/components/FormField/MapFormik/MapFormik.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, useState } from 'react';
+import { GoogleMap, LoadScript, Marker } from '@react-google-maps/api';
+
+import { FieldProps } from 'formik';
+import { containerMapStyle } from './styled';
+
+interface IProps extends FieldProps {
+    coordinates: {
+        lat: number;
+        lng: number;
+    };
+}
```

Comment:
`{
        lat: number;
        lng: number;
    }`
    It’s a separate interface